llvm/lib/Target/AMDGPU/AMDGPUHSAMetadataStreamer.cpp

```diff
@@ -663,7 +664,15 @@ void MetadataStreamerMsgPackV5::emitHiddenKernelArgs(
     Offset += 8; // Skipped.
   }
 
-  Offset += 72; // Reserved.
+  // emit argument for hidden dynamic lds size
```

Comment:
Capitalize. Also doesn't seem worth iterating over all globals

Reply:
I could find only this approach to check if dynamic LDS is used by kernel. Please let me know if there is any other way?